src/rebar_plugins.erl

```diff
@@ -39,13 +39,17 @@ project_apps_install(State) ->
     Profiles = rebar_state:current_profiles(State),
     ProjectApps = rebar_state:project_apps(State),
     lists:foldl(fun(Profile, StateAcc) ->
-                        Plugins = rebar_state:get(State, {plugins, Profile}, []),
-                        StateAcc1 = handle_plugins(Profile, Plugins, StateAcc),
+                        StateAcc1 = case Profile of
+                                        default ->
+                                            %% default profile state plugins installed in run_aux
```

Comment:
for all apps or only top-level ones?

Reply:
Yea, that is what I meant by "state" instead of "app". I can make it clearer.